src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -15,7 +15,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <configurator>
-  <bitmap name="ScenesCopyMode" type="bitmap8">
+  <bitmap name="NameSupportBitmap" type="BITMAP8">
+      <cluster code="0x0005"/>
+      <field name="SceneNames" mask="0x80"/>
+  </bitmap>
+
+  <bitmap name="ScenesCopyMode" type="BITMAP8">
```

Comment:
It seems like the only other change for Scenes, introduced by https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7495 is ScenesCopyMode (or CopyModeMap which is was originally named in spec) has also been renamed to CopyModeBitmap, so it would be great to address this in this PR as well?

Reply:
Now using CopyModeBitmap.